test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/AttributeRoutingTest.cs

```diff
@@ -180,8 +177,7 @@ private static IServiceProvider CreateServices(params ActionDescriptor[] actions
 
             var services = new ServiceCollection()
                 .AddSingleton<IInlineConstraintResolver>(new DefaultInlineConstraintResolver(routeOptions.Object))
-                .AddSingleton<ObjectPoolProvider, DefaultObjectPoolProvider>()
-                .AddSingleton<UrlEncoder>(new UrlTestEncoder());
+                .AddSingleton<ObjectPoolProvider, DefaultObjectPoolProvider>();
```

Comment:
No tests from this class were failing and it doesn't matter to Routing what's in DI. Why change this class?

Reply:
:ok: on this one removal

Reply:
Actually, :ok: on including all but the `MvcEncodedTestFixture<T>` DI changes.